app/main.py

```diff
@@ -2,24 +2,70 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.public_name = name
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: object) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"{self.public_name} must be int")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(
+                f"{self.public_name} must be between "
+                f"{self.min_amount} and {self.max_amount}"
+            )
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
```

Comment:
The `SlideLimitationValidator` class is defined as an abstract class (indicated by inheriting from `ABC`), but it does not define any abstract methods. Consider adding abstract methods to enforce implementation in subclasses.